packages/twenty-server/src/engine/core-modules/health/health-cache.service.ts

```diff
@@ -130,4 +140,37 @@ export class HealthCacheService {
 
     return aggregatedCounter;
   }
+
+  async getSystemStatus(): Promise<HealthSystem> {
```

Comment:
I think I would also but this in the admin panel but let me know if you disagree, not a very strong conviction. If we leave this here then we should ignore my other comment and keep HealthSystem there